.github/workflows/android.yml

```
@@ -0,0 +1,72 @@
+        env:
+          # Specify which environment to use for each variant
+          saksham_staging: SAKSHAM_STAG
+
+      - name: Build with Gradle
+        run: ./gradlew build
+
+      - name: Build apk project (APK) - ${{ env.main_project_module }} module
+        run: ./gradlew assemble${{ matrix.variant }}
+
+      - name: Upload APK - ${{ env.repository_name }}
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.4)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)